### PR TITLE
US - 1.2.0 - Edit Account

### PR DESCRIPTION
**Pull request by med-amine-bj**

### CODE REVIEW

File: Usiqee/Module/Account/AccountSettingsVC.swift [hunk 1]
```diff
@@ -41,30 +47,61 @@ class AccountSettingsVC: UIViewController {
         setupView()
     }
     
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
```
Comment (guaire94) [inline Usiqee/Module/Account/AccountSettingsVC.swift:51]: wa tu me crois jai jamais call le super de cet fonction mdr

File: Usiqee/Module/Account/AccountUpdateProfileVC.swift [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        ServiceAuth.updateProfile(username: username, avatar: avatar)
+        navigationController?.popViewController(animated: true)
+        ManagerAuth.shared.updateUser(username: username, avatar: avatar)
+        delegate?.didUpdateProfile()
+    }
+    
+    private func upload(image: UIImage, completion: @escaping (URL?) -> Void) {
+        guard let userId = Auth.auth().currentUser?.uid,
+            let scaledImage = image.scaledToSafeUploadSize,
+            let data = scaledImage.jpegData(compressionQuality: 0.4) else {
```
Comment (guaire94) [inline Usiqee/Module/Account/AccountUpdateProfileVC.swift:82]: In UsiqeeAdmin we have in Config file
` static let jpegCompressionQuality: CGFloat = 1.0`
Let's create the same in Usiqee if not exist, and also be aware client can see when image isn't good quality so maybe we can ajust it to 0.6 or 0.7

File: Usiqee/Module/Account/AccountSettingsVC.swift (hunk 1 above)
Comment (med-amine-bj) [reply Usiqee/Module/Account/AccountSettingsVC.swift:51]: mdr moi je rajoute `super` systematiquement quand il ya un override mais j'ai checké, c'est useless en vrai `the default implementation of this method does nothing`la doc d'apple lol

Comment (guaire94) [reply Usiqee/Module/Account/AccountSettingsVC.swift]: lolll

**Pull request merged**